src/SFA.DAS.ApplyService.Web/Views/Roatp/ProviderAlreadyOnRegister.cshtml

```diff
@@ -21,7 +21,12 @@
                     </h2>
                     <div class="govuk-error-summary__body">
                         <ul class="govuk-list govuk-error-summary__list">
-                            <li><a sfa-validation-for="ChangeApplicationRoute"></a></li>
+                            @foreach (var error in Model.ErrorMessages)
```

Comment:
What warranted this change?

Reply:
Validation stopped working, in that the top message wasn't being written.  I tried various methods to get it working using the current setup, but none worked, so went with this pattern (which is also present in various preamble pages)